### ISSUE TITLE
Periodic boundary condition option seems deprecated

### ISSUE THREAD

**Issue by RafaelMostert** [HITS-AIN/PINK#31]
The periodic boundary condition flag --pbc gets displayed both in the help message and at runtime `Use periodic boundary conditions = 1`.

Unlike in Pink version 0.23, running with or without the pbc flag makes no difference. The flag does not seem to propagate in the code beyond `src/UtilitiesLib/InputData.h`, is it deprecated?

**Comment by BerndDoser** [comment]
Yes, good point. PBC is not supported at the moment. I will add a corresponding check.
